agent/06_agent_create_cluster.sh

```diff
@@ -95,6 +95,24 @@ function create_agent_iso_no_registry() {
   popd
 }
 
+function assert_agent_no_registry_iso_size(){
+  base_dir=$SCRIPTDIR/$OCP_DIR
+  iso_name="agent-ove.$(uname -p).iso"
+  agent_iso_no_registry=$(find "$base_dir" -type f -name "$iso_name" 2>/dev/null | head -n 1)
+  iso_size=$(stat -c%s "$agent_iso_no_registry")
+  
+  # With 4.19 tech preview, the expected ISO size is approximately 36GB
+  iso_threshold=36
```

Comment:
Since this is just a basic sanity check, it won't hurt to support some minor adjustments, so I'd set the threshold size to a value slightly bigger, ie 40Gb